docs/pages/connect-your-client/tsh.mdx

```diff
@@ -952,6 +952,12 @@ the `--format` flag of `tsh play`:
 | `json` | Servers, Kubernetes clusters, applications, databases | `tsh` prints a JSON-serialized list of audit events, separated by newlines. |
 | `yaml` | Servers, Kubernetes clusters, applications, databases | `tsh` prints a YAML-serialized list of audit events, separated by `---` characters. |
 
+The `tsh play` command can also be used to speed up recordings for SSH and Kuberntes.
```

Comment:
```suggestion
The `tsh play` command can also be used to speed up recordings for SSH and Kubernetes.
```

Reply:
Ooops sorry, I meant to leave my own comment, not edit Gus's 😇